src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random;
+
+    public ColorSupplier() {
+        random = new Random();
+    }
```

Comment:
Better to initialize it right in field or create constructor with params and accept it from top-level comonents.